README.md

```diff
@@ -48,55 +44,86 @@ Formworks is a framework built on UIKit for building forms from JSON files on iO
+
+### Parameters in a Base Component
+Parameter | Type | Description | Required | Default Value
+------------ | ------------- | ------------- | ------------- | ---------
+title | String | Component's title. It should be a definition about how the field could be filled. | Yes | -
+description | String | Component's description. It could be an aditional explanation about how the field could be filled. | No | ""
+required | Bool | Specifies if the field has to be filled or not.| No | false
+validator | Validator | Determines the type of validation.| Yes | -
+
+### Parameters in a Text Component
+These parameters are in addition to the parameters in the base components.
+Parameter | Type | Description | Required | Default Value
+------------ | ------------- | ------------- | ------------- | ---------
+placeholder | String | Component's title. It should be a definition about how the field could be filled. | No | ""
+isMultiline | Bool | Component's description. It could be an aditional explanation about how the field could be filled. | No | false
+
+### Parameters in a Select Component
+These parameters are in addition to the parameters in the base components.
+Parameter | Type | Description | Required | Default Value
+------------ | ------------- | ------------- | ------------- | ---------
+options | [String] | All the available options. | Yes | -
+
+### JSON Input Example 
 ```json
-{
-    "title": "Formworks Research",
-    "components": [
-        {
-            "title": "What is your name?",
-            "componentType": "plain_text",
-            "errorMessage": "This field is obligatory. Please, answer it.",
-            "required": true,
-            "specs": {
-                "placeholder": "Type your name here."
+    {
+        "id": "87986E91-247F-4F36-A577-19DF6BD165D0",
+        "responseType": "long",
+        "title": "Formworks Title",
+        "components": [
+            {
+                "id": "87986E91-247F-4F36-A577-19DF6BD165D0",
+                "type": "text",
+                "title": "What is your name?",
+                "description": "Type your name.",
+                "required": true,
+                "validator": "max32",
+                "placeholder": "Your name",
+                "isMultiline": false
+            },
+            {
+                "id": "87986E91-247F-4F36-A577-19DF6BD165D0",
+                "type": "text",
+                "title": "What is your e-mail?",
+                "description": "Type your e-mail.",
+                "required": true,
+                "validator": "email",
+                "placeholder": "youremail@example.org",
+                "isMultiline": false
+            },
+            {
+                "id": "87986E91-247F-4F36-A577-19DF6BD165D0",
+                "type": "text",
+                "title": "Tell us a little bit about yourself",
+                "description": "We want to know more about you.",
+                "validator": "max256",
```

Comment:
The max256 validator does not exist. The correct validator is max32. @cassiaabarbosa